com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py

```diff
@@ -1220,4 +1427,48 @@ def get_instance_status(self):
             dict: JSON response for the instance status operation.
         """
         status_url = self._get_url('status_path')
-        return self.rest_client.session.get(status_url).json()
+        res = self.rest_client.session.get(status_url)
+        self.rest_client.handle_http_errors(res)
+        return res.json()
+
+class IAMConstants(object):
```

Comment:
Do we want this to be a public class or should it be `_IAMConstants` ??

Reply:
There are cases where we use the constants outside of `rest_primitives`. For example, the canonical test for `v2` is 
```
if IAMConstants.V2_REST_URL in credentials:
    ...
```

This code is used in our test classes.

Reply:
Doesn't need to be a class we expose though, it can still be `_IAMConstants` and used by the `streamsx` package internally.